module/VuFind/src/VuFind/ILS/Driver/Folio.php

```diff
@@ -282,15 +289,30 @@ public function preRequest(\Laminas\Http\Headers $headers, $params)
      */
     protected function renewTenantToken()
     {
+        // If not using legacy authentication, see if the token has expired before trying to renew it
+        if (! $this->useLegacyAuthentication() && ! $this->checkTenantTokenExpired()) {
```

Comment:
Trivial style change: we don't typically include spaces after the `!` operator (though apparently we don't have an automatic rule set to normalize this).

```suggestion
        if (!$this->useLegacyAuthentication() && !$this->checkTenantTokenExpired()) {
```

Reply:
Done. I couldn't recall if we liked the space or not -- I'll try to remember for next time.